docs/storagenetwork/storagenetwork.md

```diff
@@ -0,0 +1,239 @@
+- All VMs should be stopped.
+    - We recommend checking the VM status with the following command and should get an empty result.
+    - `kubectl get -A vmi`
+- All pods that are attached to Longhorn Volumes should be stopped.
+    - Users could skip this step with the Harvester Storage Network setting. Harvester will stop Longhorn-related pods automatically.
+
+## Configuration Example
+
+- IP range for Storage Network
+	- IP range should not conflict or overlap with Kubernetes cluster networks.
+	- IP range should be IPv4 CIDR format and 4 times the number of your cluster nodes. Longhorn will use 2 IPs per node and the upgrade process will run two versions of Longhorn simultaneously. It will consume 4 IPs per node in the upgrade process.
+	- If your cluster has 250 nodes, the IP range should be larger than /22.
+- Well-configured Cluster Network and VLAN Config
+	- Please refer Networking page for more details and configure `Cluster Network` and `VLAN Config` but not `Networks`.
+- VLAN ID
+	- Please check with your network switch setting, and provide a dedicated VLAN ID for Storage Network
+
+We will take the following configuration as an example to explain the details of the Storage Network
+
+- IP range: `192.168.0.0/24`
+- Cluster Network: `storage`
+- VLAN ID for Storage Network: `100`
+
+## Configuration Process
+
+Harvester will create Multus NetworkAttachmentDefintion from the configuration, stop pods related to Longhorn Volume, update Longhorn setting, and restart previous pods.
+
+### Before Applying Harvester Storage Network Setting
+
+Here we have two cases.
+- Expect that VM VLAN traffic and Longhorn Storage Network use the same group of physical interfaces.
+- Expect that VM VLAN traffic and Longhorn Storage Network use different physical interfaces.
+
+Longhorn will send replication traffic through the specific interfaces shown as the red line in the figure.
+
+#### Same Physical Interfaces
+
+Take `eth2` and `eth3` as an example for VM VLAN traffic and Longhorn Storage Network simultaneously.
+
+Please refer Networking page to configure `ClusterNetwork` and `VLAN Config` with `eth2` and `eth3` and remember the `ClusterNetwork` name for the further step.
+
+![storagenetwork-same.png](./storagenetwork-same.png)
+
+
+#### Different Physical Interfaces
+
+`eth2` and `eth3` are for VM VLAN Traffic. `eth4` and `eth5` are for Longhorn Storage Network.
+
+Please refer Networking page to configure `ClusterNetwork` and `VLAN Config` with `eth4` and `eth5` for Storage Network and remember the `ClusterNetwork` name for the further step.
+
+![storagenetwork-diff.png](./storagenetwork-diff.png)
+
+### Harvester Storage Network Setting
+
+Harvester Storage Network setting will need `range`, `clusterNetwork`, `vlan` field to construct Multus NetworkAttachmentDefintion for Storage Network usage. You could apply this setting via Web UI or CLI.
```

Comment:
Can we standardize the capitalization of clusterNetwork? It should be either ClusterNetwork or clusterNetwork or clusternetwork.

Reply:
in the setting, "clusterNetwork" here is the JSON entry name that is case-sensitive.
Other parts will be the concept name or CR name